citylearn/electric_vehicle_charger.py

```diff
@@ -0,0 +1,374 @@
+import inspect
+from typing import List, Dict
+from citylearn.base import Environment
+
+from citylearn.electric_vehicle import electric_vehicle
+
+ZERO_DIVISION_CAPACITY = 0.00001
+
+
+class Charger(Environment):
+    def __init__(
+            self,
+            nominal_power: float,
+            efficiency: float = None,
+            charger_id: str = None,
+            charger_type: int = None,
+            max_charging_power: float = 50.0,
+            min_charging_power: float = 0.0,
+            max_discharging_power: float = 50.0,
+            min_discharging_power: float = 0.0,
+            charge_efficiency_curve: Dict[float, float] = None,
+            discharge_efficiency_curve: Dict[float, float] = None,
+            image_path: str = None,
+            connected_ev: electric_vehicle = None, incoming_ev: electric_vehicle = None,
```

Comment:
There seems to be inconsistency with how electric vehicles are named in this module. It is called `car`, `ev`, and `electric_vehicle` at different points. Update all to use the same name; `electric_vehicle` preferrably.